packages/components/header/src/header.css

```diff
@@ -7,13 +7,15 @@
 		width: 100vw;
 		height: var(--ts-unit-triple);
 		background-color: var(--ts-color-white);
+		border-bottom: var(--ts-border);
 		display: flex;
 		align-items: center;
 		flex-direction: row;
 	}
 
 	& .icon {
 		margin-left: var(--ts-unit-half);
+		block-size: var(--ts-unit-double);
```

Comment:
What was the reason to choose `block-size`? IE and Edge not supported and I can't see any difference with or without them. 
https://developer.mozilla.org/en-US/docs/Web/CSS/block-size

Reply:
The browser add some block-size make the icon not vertically center.